src/Entity/User/AdminUser.php

```diff
@@ -21,22 +21,26 @@
 class AdminUser extends BaseUser implements AdminUserInterface
 {
     /**
-     * @var string
+     * @var string|null
      *
      * @ORM\Column(type="string", nullable=true)
      */
-    protected $lastName;
+    private $lastName;
 
     /**
-     * @var string
+     * @var string|null
      *
      * @ORM\Column(type="string", nullable=true)
      */
-    protected $firstName;
+    private $firstName;
 
     /**
-     * AdminUser constructor.
+     * @var AdminAvatar|null
+     *
+     * @ORM\OneToOne(targetEntity="App\Entity\User\AdminAvatar", cascade={"persist"})
```

Comment:
Shouldn't we introduce an interface for AdminAvatar as well? In end projects, I'm ok with entities without an interface, but it is more about being coherent with rest of the classes.

Reply:
I added an interface for extended entities (from Sylius). But I think you're right, I could use an interface for all entities, it's indeed more coherent.